index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+
+    <header>
+        <nav id="phone">
+            <div class="logo">My Logo</div>
+            <img src="images/humbarger.svg" alt="humbarger" class="humbarger">
+        </nav>
+    </header>
+    <main>
+
+        <section id="bg">
+            <section id="bio">
+                <h1 class="title">I’m Mariane Glad to see you!</h1>
+                <article class="about">I’m a software developer! I can help you build a product , feature or website
+                    Look
+                    through
+                    some
+                    of my work and
+                    experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+                    me.
+                </article>
+                <div class="social-title">LET’S CONNECT</div>
+                <div class="social">
+                    <img src="images/social/twitter.svg" alt="twitter">
+                    <img src="images/social/linkedin.svg" alt="linkedin">
+                    <img src="images/social/m.svg" alt="m">
+                    <img src="images/social/github.svg" alt="github">
+                    <img src="images/social/hand.svg" alt="hand">
+
+                </div>
```

Comment:
As per the requirement, the list of icons should use a `ul` tag to group them, and `li` per icon; each icon should have a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.
- [ ] How about changing the `<div>` to `<ul>`and wrap the `<img>` inside `<li><a href="..."><img .../><a></li>` as per the requirement? 💯 😉 
```
<ul class="social">
   <li>
        <a href="...">
            <img src="images/social/twitter.svg" alt="twitter">
         </a>
   </li>
   ...
</ul>
```

- [ ] Please test the UI and make the changes in the CSS if required. 🟢 💯